.github/workflows/bazel.yml

```diff
@@ -73,48 +72,6 @@ jobs:
       - name: Check Out Repo
         # This is necessary for overlays into the Docker container below.
         uses: actions/checkout@v2
-      - name: Bazel Cache
-        uses: actions/cache@v2
-        with:
-          path: ${{ github.workspace }}/.${{ env.BAZEL_CACHE }}
-          key: ${{ runner.os }}-${{ env.BAZEL_CACHE }}-${{ env.CACHE_SUB_KEY_BUILD_ALL }}-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-${{ env.BAZEL_CACHE }}-${{ env.CACHE_SUB_KEY_BUILD_ALL }}-
-
-      - name: Bazel Cache Repo
```

Comment:
Didn't we want to keep the repo cache? Anyway, I'm also fine with removing it if that's not too bad for performance.

Reply:
At least with the bazel version that we are using (4.2.2) it does not seem possible to combine the repository cache with a remote cache. See open feature request https://github.com/bazelbuild/bazel/issues/6359. We also tested the `--experimental_remote_downloader` option, which did not work either in our setup.

Reply:
We cannot use the remote cache as a repository cache, but can we use `--repository_cache` to have an on-disk repository cache as before?

Reply:
In bazel 4.2.2 it seems that the local `repository_cache` is ignored if a remote cache is present, same as for the disk cache.